run_docker.sh

```diff
@@ -1 +0,0 @@
-docker run -ti --rm -v $(pwd):/Qt.py --entrypoint bash mottosso/qt.py27
```

Comment:
Ah, sorry. The `--entrypoint` overrides the `ENTRYPOINT` from the image, dropping you into a bash shell where you can run Python and friends interactively.

That is the full contents of the file, you can just replace the image name `mottosso/qt.py27` with the new one.